src/reusable.ts

```diff
@@ -3,17 +3,18 @@ export type StoreValueChangeCallback<HookValue> = (value: HookValue | null) => v
 export type StoresChangeCallback = () => void;
 
 export class Store<HookValue> {
-  name: string = 'Store';
+  name: string;
   subscribers: StoreValueChangeCallback<HookValue>[] = [];
   cachedValue: HookValue | null = null;
   constructor(private fn: HookFn<HookValue>) {
+    this.name = fn.name || 'Store';
   }
 
   getValue() {
     return this.cachedValue;
   }
 
-  run() {
+  value() {
```

Comment:
having both getValue and value on the store is very confusing. maybe it would be better called as computeValue and getValue will be getCachedValue.

Reply:
value => useValue.
getValue => getCachedValue